.github/workflows/ci.yml

```diff
@@ -13,6 +13,15 @@ jobs:
   build:
     # Use new enough OS that has CGroupsv2 enabled (required by the integration tests)
     runs-on: ubuntu-22.04
+    env:
+      TEST_DATABASE_URL: postgresql://postgres:postgres@localhost:5432/postgres?sslmode=disable
+    services:
+      postgres:
+        image: postgres:14
```

Comment:
Using 14 as a minimum here since the EXPLAIN ANALYZE tests rely on being able to turn off `compute_query_id` in the output (and that was added in 14).